code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -835,6 +879,8 @@
 			/obj/machinery/factory/former = 1,
 			/obj/machinery/factory/reconstructor = 1,
 			/obj/machinery/unboxer = 1,
+			/obj/item/stack/conveyor/thirty = -1,
+			/obj/item/conveyor_switch_construct = -1,
```

Comment:
this is the real vendor, not the valhalla one

Reply:
![image](https://user-images.githubusercontent.com/66163761/223895271-704c2f1b-324a-4545-b1e2-94cd815510f8.png)
Is 10 of each ok?